x-pack/plugins/security_solution/public/timelines/components/timeline/query_tab_content/query_tab_unified_components.test.tsx

```diff
@@ -0,0 +1,615 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ComponentProps } from 'react';
+import React, { useEffect } from 'react';
+import QueryTabContent from '.';
+import { defaultRowRenderers } from '../body/renderers';
+import { TimelineId } from '../../../../../common/types/timeline';
+import { useTimelineEvents } from '../../../containers';
+import { useTimelineEventsDetails } from '../../../containers/details';
+import { useSourcererDataView } from '../../../../common/containers/sourcerer';
+import { mockSourcererScope } from '../../../../common/containers/sourcerer/mocks';
+import { mockTimelineData, TestProviders } from '../../../../common/mock';
+import { DefaultCellRenderer } from '../cell_rendering/default_cell_renderer';
+import { render, screen, waitFor, fireEvent, within } from '@testing-library/react';
+import { createStartServicesMock } from '../../../../common/lib/kibana/kibana_react.mock';
+import type { StartServices } from '../../../../types';
+import { useKibana } from '../../../../common/lib/kibana';
+import { useDispatch } from 'react-redux';
+import { timelineActions } from '../../../store';
+import type { ExperimentalFeatures } from '../../../../../common';
+import { allowedExperimentalValues } from '../../../../../common';
+import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
+import { cloneDeep } from 'lodash';
```

Comment:
As an aside, I wonder if we can remove these in favor of: https://developer.mozilla.org/en-US/docs/Web/API/structuredClone

Reply:
makes sense... Changing them... one by one :D